### PR TITLE
add com.github.skullernet.q2pro

### PR DESCRIPTION
**Pull request by fishxz**
a modern quake2 client focused on multiplayer

### CODE REVIEW

File: com.github.skullernet.q2pro.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - install -Dm644 com.github.skullernet.q2pro.appdata.xml -t /app/share/metainfo
+      - install -Dm644 src/unix/res/q2pro.desktop -t /app/share/applications
+      - install -Dm644 q2pro.svg -t /app/share/icons/hicolor/scalable/apps
+    sources:
+      - type: git
+        url: https://github.com/skullernet/q2pro.git
+        tag: r1826
+        commit: 7ad132affeba68a91984852851630b2fe417c307
+      - type: file
+        path: com.github.skullernet.q2pro.appdata.xml
```
Comment (LeandroStanger) [inline com.github.skullernet.q2pro.yaml:44]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by fishxz** [comment]
bot, build com.github.skullernet.q2pro

**Comment by fishxz** [comment]
bot, build com.github.skullernet.q2pro

**Comment by fishxz** [comment]
bot, build com.github.skullernet.q2pro

**Comment by fishxz** [comment]
bot, build com.github.skullernet.q2pro

**Comment by bilelmoussaoui** [comment]
Have you reached out to upstream developers if they were interested in (co-)maintain their game on Flathub?

**Comment by fishxz** [comment]
> Have you reached out to upstream developers if they were interested in (co-)maintain their game on Flathub?

https://github.com/skullernet/q2pro/issues/186

**Comment by Eonfge** [comment]
bot, build com.github.skullernet.q2pro

**Comment by Eonfge** [comment]
I've tested it with the proprietary assets and I can confirm that both single and multiplayer work. And that I suck in Q2 DM, but let's not dwell on that. 

I do have one serious complaint. The logo is not the same as that of Q2PRO. I assume that you've taken the logo from the Yamagi package (whom I maintain), but the actual Q2PRO logo is different, it's dark green:

https://github.com/skullernet/q2pro/blob/master/src/unix/res/q2pro.xpm

Could you take the Yamagi logo and make it dark green, this will make them both look different.

**Comment by fishxz** [comment]
> I've tested it with the proprietary assets and I can confirm that both single and multiplayer work. And that I suck in Q2 DM, but let's not dwell on that.
> 
> I do have one serious complaint. The logo is not the same as that of Q2PRO. I assume that you've taken the logo from the Yamagi package (whom I maintain), but the actual Q2PRO logo is different, it's dark green:
> 
> https://github.com/skullernet/q2pro/blob/master/src/unix/res/q2pro.xpm
> 
> Could you take the Yamagi logo and make it dark green? This will help users in distinguishing the different packages.
> 
> Addendum:
> Just to help you on your way, Dark Green is actually `#006400` according to the X11 colour names:
> https://en.wikipedia.org/wiki/X11_color_names

hey,
thanks for the feedback. the reason why i dont use the q2pro logo, is because of the very low resolution.

i changed the color of the yamagi logo to fit q2pro.

**Comment by fishxz** [comment]
bot, build com.github.skullernet.q2pro

**Comment by fishxz** [comment]
Well, I asked upstream over one year ago. 

What is missing?

**Comment by nedrichards** [comment]
hah! that I misread 2019 as 2020 ;-)

**Pull request closed without merging**